src/pyenphase/auth.py

```
@@ -3,16 +3,37 @@
+import aiohttp
 import jwt
 import orjson
+from aiohttp import ClientHandlerType, ClientMiddlewareType
 from tenacity import retry, retry_if_exception_type, wait_random_exponential
 
 from .const import LOCAL_TIMEOUT, URL_AUTH_CHECK_JWT
 from .exceptions import EnvoyAuthenticationError, EnvoyAuthenticationRequired
 from .ssl import SSL_CONTEXT
 
 
+class CloseConnectionOnAuthRequiredMiddleware:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)